ci/azure-pipelines.yml

```diff
@@ -8,18 +8,18 @@ trigger:
 
 variables:
   GOPATH: $(Agent.BuildDirectory)/go
-  GOVER: 1.11.5
+  GOVER: 1.14.3
   PATH: $(Agent.BuildDirectory)/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
 
 pool:
-  vmImage: ubuntu-16.04
+  vmImage: ubuntu-18.04
 
 steps:
 - task: GoTool@0
   inputs:
     version: $(GOVER)
     goPath:  $(GOPATH)
-  displayName: Install GoLang
+  displayName: Install Go
 - checkout: self
   path: 'go/src/github.com/hyperledger/fabric-lib-go'
```

Comment:
As a module, the checkout should be moved outside of the GOPATH to help ensure it really is a standalone module.